### ISSUE TITLE
Is the loss ok or not?

### ISSUE THREAD

**Issue by BlueAnthony** [Zhenye-Na/image-similarity-using-deep-ranking#6]
mini Batch Loss: 0.838141679763794
mini Batch Loss: 0.038405612111091614
mini Batch Loss: 0.28175681829452515
mini Batch Loss: 1.1555051803588867
mini Batch Loss: 1.0039609670639038
mini Batch Loss: 0.43094536662101746
mini Batch Loss: 1.021866798400879
mini Batch Loss: 0.7752935886383057
mini Batch Loss: 0.23081330955028534
mini Batch Loss: 0.0
mini Batch Loss: 0.24504965543746948
mini Batch Loss: 0.8028221130371094
mini Batch Loss: 0.511269748210907
mini Batch Loss: 0.6132020354270935
mini Batch Loss: 1.1050782203674316
mini Batch Loss: 0.0
mini Batch Loss: 0.0
mini Batch Loss: 0.016318656504154205
mini Batch Loss: 0.0
mini Batch Loss: 0.0
mini Batch Loss: 0.014867536723613739
mini Batch Loss: 0.6881022453308105
mini Batch Loss: 0.0
mini Batch Loss: 0.08096975088119507
mini Batch Loss: 0.0
mini Batch Loss: 0.0
mini Batch Loss: 0.05993702635169029
mini Batch Loss: 0.8831809759140015
mini Batch Loss: 0.0
mini Batch Loss: 0.5886887311935425
mini Batch Loss: 0.241916224360466
mini Batch Loss: 0.29788437485694885
mini Batch Loss: 0.0
mini Batch Loss: 0.0
mini Batch Loss: 0.00650769891217351
mini Batch Loss: 1.1349914073944092
mini Batch Loss: 0.13306227326393127
mini Batch Loss: 0.0
![loss](https://user-images.githubusercontent.com/17959032/55785722-86609d00-5ae5-11e9-8e30-da3d151cf33b.jpeg)

I use the default setting and provided triplets.txt to fine-tune the pretrained model of resnet34 from torchvision.
But as figure above, there are a lot of zeros. I'm not sure if it is normal that the loss start from 3, as I see your loss visualization in README.md which start around 0.9.
And loss distribution is not that smooth as yours. 
Any suggestion?
Thank you!!
